deploy-manage/license.md

```diff
@@ -1,11 +1,31 @@
 ---
+applies_to:
 mapped_pages:
   - https://www.elastic.co/guide/en/cloud/current/ec-licensing.html
```

Comment:
maybe https://docs-v3-preview.elastic.dev/elastic/docs-content/pull/539/deploy-manage/cloud-organization/billing/manage-subscription is a better target for this mapping. wdyt? 

also please re-add this sentence to this page:
> You are entitled to use all of the features in Elastic that match your subscription level. Please use them to your heart’s content. 😍